src/parsers/texttracks/ttml/html/ttml_color_to_css_color.ts

```diff
@@ -35,14 +38,31 @@ export default function ttmlColorToCSSColor(color : string) : string {
       String(parseInt(regRes[3], 16)) + "," +
       String(parseInt(regRes[4], 16) / 255) + ")";
   }
-  regRes = REGXP_4_HEX_COLOR.exec(color);
 
+  regRes = REGXP_4_HEX_COLOR.exec(color);
   if (regRes != null) {
     return "rgba(" +
       String(parseInt(regRes[1] + regRes[1], 16)) + "," +
       String(parseInt(regRes[2] + regRes[2], 16)) + "," +
       String(parseInt(regRes[3] + regRes[3], 16)) + "," +
       String(parseInt(regRes[4] + regRes[4], 16) / 255) + ")";
   }
+
+  regRes = REGXP_RGB_COLOR.exec(color);
+  if (regRes != null) {
+    return "rgb(" +
```

Comment:
I don't get the point of returning this. Isn't `color` the same thing ?

Reply:
Well, is it really possible to have spaces between ?

Reply:
it is. But I thought the code looked clearer that way

Reply:
ok